src/main/host/host.rs

```diff
@@ -814,6 +824,10 @@ impl Drop for Host {
             unsafe { cshadow::tracker_free(tracker.ptr()) };
         };
 
+        let futex_table = self.futex_table.borrow().ptr();
```

Comment:
I think should be `borrow_mut()` since it mutates the ref count and frees the table.

Reply:
done